contractcourt/chain_watcher.go

```diff
@@ -974,24 +977,62 @@ func (c *chainWatcher) handleUnknownRemoteState(
 // return zero. This is possible as our output may have been trimmed due to
 // being dust.
 func (c *chainWatcher) toSelfAmount(tx *wire.MsgTx) btcutil.Amount {
-	var selfAmt btcutil.Amount
-	for _, txOut := range tx.TxOut {
+	// There are two main cases we have to handle here. First, in the coop
+	// close case we will always have saved the delivery address we used
+	// whether it was from the upfront shutdown, from the delivery address
+	// requested at close time, or even an automatically generated one. All
+	// coop-close cases can be identified in the following manner:
```

Comment:
Why does this end with a colon?

Reply:
to indicate that the immediately following code implements the stated logic.